README.md

```diff
@@ -35,7 +40,9 @@ var add = function (x, y) {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5
+not anonymous
+add()
```

Comment:
This is anonymous. We call this a **function expression**. Where a function is assigned to a variable. Because the function has no name, it's anonymous